peter-piper.js

```diff
@@ -13,13 +13,26 @@
 
 var peterpiper = '';
 peterpiper += "Peter Piper picked a peck of pickled peppers.";
-peterpiper += "A peck of pickled peppers Peter Piper picked."
-peterpiper += "If Peter Piper picked a peck of pickled peppers,"
-peterpiper += "Where's the peck of pickled peppers Peter Piper picked?"
+peterpiper += "A peck of pickled peppers Peter Piper picked.";
+peterpiper += "If Peter Piper picked a peck, peck of pickled peppers,";
+peterpiper += "Where's the peck of pickled peppers peck? Peter Piper picked?";
 
-var counter = 0;
+function countOccurrences(string, word){
+	var counter = 0;
+	if (word.length > string.length) {
+		return 0;
+	} else {
+		var strings = string.replace(/[.,-\/#!$%\^&\*;:{}=\-_`~()?]/g," ").split(' ');
```

Comment:
You might consider using the `\W` character class shorthand, which stands for any character that that is not `[A-Za-z0-9_]`: http://www.regular-expressions.info/shorthand.html.

The lowercase `\w` matches any "word character", or the characters `[A-Za-z0-9_]`.

The uppercase `\W` matches any character that's NOT a word character.